src/noit_check_log.c

```diff
@@ -684,6 +752,8 @@ noit_check_log_bundle_serialize(mtev_log_stream_t ls, noit_check_t *check) {
     // Compress + B64
     comp = use_compression ? NOIT_COMPRESS_ZLIB : NOIT_COMPRESS_NONE;
     noit_check_log_bundle_compress_b64(comp, buf, size, &out_buf, &out_size);
+    mtevL(mtev_debug, "B%c compression: %f%%\n", use_compression ? '1' : '2',
```

Comment:
out_size could be undefined (never initialized), there are three return paths in noit_check_log_bundle_compress_b64() that don't set out_buf or out_size at all.